cumulusci.yml

```diff
@@ -179,7 +182,12 @@ tasks:
                 - fullName: "Walking to or from School"
                   label: "Walking to or from School"
                   add_before: "Other"
-
+                  
+    add_citizenship_status_values:
+        description: Runs an EDA task that adds additioanl picklist value to hed__Citizenship_Status__c field. 
+        steps:
+            1. task: eda:add_citizenship_status_values
+            
```

Comment:
This is a task calling a task, which will not function in CumulusCI. Tasks call Python Classes.
```suggestion
```